SMLHelper/Patchers/LanguagePatcher.cs

```diff
@@ -15,12 +16,17 @@ internal class LanguagePatcher
         private const char TextDelimiterOpen = '{';
         private const char TextDelimiterClose = '}';
         private const char KeyValueSeparator = ':';
```

Comment:
What if the user forgets to close the `}`? With this PR, there's no real need to keep the text delimiters. At least not both of them.

Reply:
Forgetting the end delimiter would break multi-line values.  
Not really an option.